### PR TITLE
KDE ISO Image Writer package

### PR DESCRIPTION
**Pull request by jriddell**
### Please confirm your submission meets all the criteria

- [ X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X ] My pull request follows the instructions at [App Submission][submission].
- [X ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X ] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X ] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
so #3031 can be closed?

**Comment by hfiguiere** [comment]
cc  @flathub/kde

File: org.kde.isoimagewriter.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+   "sdk": "org.kde.Sdk",
+   "command": "isoimagewriter",
+   "finish-args": [
+      "--device=dri",
+      "--filesystem=host",
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--system-talk-name=org.freedesktop.UDisks2",
+      "--talk-name=org.freedesktop.Flatpak"
```
Comment (hfiguiere) [inline org.kde.isoimagewriter.json:14]: is this access really necessary? The previous attempt #3031 didn't require it.

Access to this name require an exception for the linter.

Also about the filesystem access: https://github.com/flathub/flathub/pull/3031/files#r852934978

File: org.kde.isoimagewriter.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+   ],
+   "modules": [
+      {
+         "buildsystem": "cmake-ninja",
+         "name": "isoimagewriter",
```
Comment (hfiguiere) [inline org.kde.isoimagewriter.json:19]:
```suggestion
         "name": "isoimagewriter",
         "buildsystem": "cmake-ninja",
```

**Comment by hfiguiere** [comment]
bot, build org.kde.isoimagewriter

File: org.kde.isoimagewriter.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+         "sources": [
+            {
+               "type": "archive",
+               "url": "https://download.kde.org/stable/isoimagewriter/1.0.0/isoimagewriter-1.0.0.tar.xz",
+               "sha256": "a69022303ec1beb6b7b4f0d68e59e4199d3c9868767cb9eb23c6dd1999e06a5d",
+               "x-checker-data": {
+                     "type": "anitya",
+                     "project-id": 8763,
```
Comment (tsdgeos) [inline org.kde.isoimagewriter.json:27]: This is wrong https://release-monitoring.org/project/8763 is not isoimagewriter 

https://release-monitoring.org/project/8763

**Comment by jriddell** [comment]
> so #3031 can be closed?

Yes

File: org.kde.isoimagewriter.json (hunk 3 above)
Comment (jriddell) [reply org.kde.isoimagewriter.json:27]: how do I find out the project-id?

File: org.kde.isoimagewriter.json (hunk 1 above)
Comment (jriddell) [reply org.kde.isoimagewriter.json:14]: The app now uses udisks2 so the udisks2 permission is necessary.  The previous attempt failed.  

Filesystem access is needed to read the ISO files to write.

File: org.kde.isoimagewriter.json (hunk 3 above)
Comment (tsdgeos) [reply org.kde.isoimagewriter.json:27]: Use the search field/button on the site.

But I can see it not existing so you have three options: 
 - Not have a x-checker-data
 - Add kde iso image writter to release-monitoring.org
 - Use a different x-checker-data backend

File: org.kde.isoimagewriter.json (hunk 1 above)
Comment (jriddell) [reply org.kde.isoimagewriter.json]: dunno, I guess not, removed

**Comment by CarlSchwan** [comment]
bot, build org.kde.isoimagewriter

File: org.kde.isoimagewriter.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+               "type": "archive",
+               "url": "https://download.kde.org/stable/isoimagewriter/1.0.0/isoimagewriter-1.0.0.tar.xz",
+               "sha256": "a69022303ec1beb6b7b4f0d68e59e4199d3c9868767cb9eb23c6dd1999e06a5d",
+               "x-checker-data": {
+                     "type": "anitya",
+                     "project-id": 341536,
+                     "stable-only": true,
+                     "url-template": "https://download.kde.org/stable/isoimagewriter/$version/isoimagewriter-$version.tar.xz"
+                  }
+            }
```
Comment (CarlSchwan) [inline org.kde.isoimagewriter.json:30]:
```suggestion
            },
            {
                    "type": "patch",
                    "path": "logo.patch"
            }
```

with logo.patch being generated with `git show 62daf9a > logo.patch`

**Comment by CarlSchwan** [comment]
bot, build org.kde.isoimagewriter

**Comment by CarlSchwan** [comment]
bot, build org.kde.isoimagewriter

**Comment by CarlSchwan** [comment]
bot, build org.kde.isoimagewriter

**Comment by jriddell** [comment]
bot, build org.kde.isoimagewriter

**Comment by jriddell** [comment]
This is running great for me

**Comment by tsdgeos** [comment]
Looks good to me

**Comment by jriddell** [comment]
What's the next step?

**Comment by barthalion** [comment]
bot, build org.kde.isoimagewriter

**Comment by barthalion** [comment]
/merge

@tsdgeos, please verify it as upstream-maintained when it becomes visible on the website. You will need to log in with both GitHub and KDE accounts at https://flathub.org/login, probably refresh the list of apps with the refresh button, and access developer settings for ISO Image Writer.

**Pull request closed without merging**